src/api/messages/sendMessage.ts

```diff
@@ -25,7 +25,7 @@ export const sendMessage = async (
     body: payload.body,
     recipient: payload.recipient,
     hops: payload.hops,
-    path: payload.path
+    path: payload.path ?? []
```

Comment:
AFAIK this will make it send direct messages all the time and the `HOPS` value will not be taken into account

Reply:
Good to know - updated accordingly

Reply:
Yeah, Steven is right.
It would always send direct messages if path is not set in `sendMessage({path})`, as hops integer is ignored if path is set In the REST API.